README.md

```diff
@@ -1,10 +1,10 @@
 # Chef
 
-A cookbook of Chef recipes for the Deep Security agent. This allows for the easy deployment of the Deep Security agent as well as taking some common actions from the agent.
+A cookbook of Chef recipes for the Trend Micro Deep Security Agent. This allows for the easy deployment of the Trend Micro Deep Security Agent as well as taking some common actions from the agent.
```

Comment:
The sentence that starts with "This allows ..." -> (rewrite to):  "The recipes allow you to easily deploy the Deep Security Agent and perform a few common agent tasks."